File: scripts/patch-discord-autobet-buttons.py
#!/usr/bin/env python3
from pathlib import Path
import os
import sys
TARGET = Path(__file__).resolve().parents[1] / 'skill' / 'discord-button-repatch' / 'scripts' / 'repatch_discord_buttons.py'
os.execvp('python3', ['python3', str(TARGET), *sys.argv[1:]])

File: scripts/patch-discord-autobet-buttons.js
#!/usr/bin/env node
const path = require('path');
const { spawnSync } = require('child_process');
const target = path.resolve(__dirname, '..', 'skill', 'discord-button-repatch', 'scripts', 'repatch_discord_buttons.py');
const result = spawnSync('python3', [target, ...process.argv.slice(2)], { stdio: 'inherit' });
if (result.error) throw result.error;
process.exit(result.status ?? 0);
